src/coordinate-grid-mixin.js

```diff
@@ -1034,28 +1087,34 @@ dc.coordinateGridMixin = function (_chart) {
         }
     };
 
-    _chart.redrawBrush = function (g, doTransition) {
-        if (_brushOn) {
-            if (_chart.filter() && _chart.brush().empty()) {
-                _chart.brush().extent(_chart.filter());
-            }
+    _chart.redrawBrush = function (selection) {
```

Comment:
Why did you remove `doTransition` here? [_brush.move](https://github.com/d3/d3-brush/blob/master/README.md#brush_move) can take a transition or a selection, just like the old brush could be applied to either a selection or transition.

It's not a big deal, just a tiny feature that makes chart resizing work better, but it makes me worry when functionality is removed with no explanation. In the future, could you please try to document the things that you disable or remove? 

I guess I need to go over this PR with a fine-toothed comb before we release 3.0. We don't have full test coverage, especially where transitions are concerned - it's unclear how to test them.

Reply:
Thanks for letting me know, I will mark in future if I remove or change any functionality.

This one, in my initial implementation, was aggravating the infinite recursion. However the current code, may not suffer from that.

I will be more comfortable if you could go through the entire change set considering it to be my first encounter with both dc and d3.

Reply:
I see, you probably simplified it in order to figure out why it was recursive, and then didn't restore the feature when you were done.

No harm done. Yes, I'll read over all the changes before we release.